src/elements/Avatar.tsx

```diff
@@ -1,37 +1,102 @@
-// @ts-ignore
-import React from "react"
+import React, { ImgHTMLAttributes, SFC } from "react"
 import styled from "styled-components"
 import { space, SpaceProps } from "styled-system"
+import { color } from "../helpers/color"
+import { SerifSize } from "../Theme"
+import { Flex } from "./Flex"
+import { Serif } from "./Typography"
 
 /**
  * Spec: TODO
  */
 
-const SIZE = { xs: "45px", sm: "70px", md: "100px" }
+interface SizeProps {
+  [key: string]: {
+    diameter: string
+    typeSize: SerifSize
+  }
+}
+
+const Size: SizeProps = {
+  xs: {
+    diameter: "45px",
+    typeSize: "3",
+  },
+  sm: {
+    diameter: "70px",
+    typeSize: "6",
+  },
+  md: {
+    diameter: "100px",
+    typeSize: "8",
+  },
+}
 
 const sizeValue = ({ size = "" }) => {
   switch (size) {
     case "xs":
-      return SIZE.xs
+      return Size.xs
     case "sm":
-      return SIZE.sm
+      return Size.sm
     case "md":
     default:
-      return SIZE.md
+      return Size.md
   }
 }
 
-export interface AvatarProps extends SpaceProps {
+export interface AvatarProps extends ImgHTMLAttributes<any>, SpaceProps {
+  /** If an image is missing, show initials instead */
+  initials?: string
   /** The size of the Avatar */
   size?: "xs" | "sm" | "md"
 }
 
+/** An circular Avatar component containing an image or initials */
+export const Avatar: SFC<AvatarProps> = ({ src, initials, size = "sm" }) => {
```

Comment:
Should not it default to `md` since it's the default for the props? Or is it changed intentionally here?

Reply:
Ohh, good catch, will update 👍